mbedtls/patches/03_avoid_relinking

```diff
@@ -0,0 +1,12 @@
+diff -ur '--exclude=*.git' mbedtls-2.28.0/Makefile mbedtls-2.28.0-phoenix/Makefile
+--- mbedtls-2.28.0/Makefile	2022-06-03 14:45:36.268697648 +0200
++++ mbedtls-2.28.0-phoenix/Makefile	2022-06-03 14:55:26.338969692 +0200
+@@ -39,7 +39,7 @@
+ 	    if [ -x $$p ] && [ ! -d $$p ] ;     \
+ 	    then                                \
+ 	        f=$(PREFIX)`basename $$p` ;     \
+-	        cp $$p $(DESTDIR)/bin/$$f ;     \
```

Comment:
no, I was talking about installing **libraries** (it's already done correctly for **headers**). Maybe I was too brief in my explanation:
 - currently mbedtls will **always** update installed library files modification time - even if the library itself was not rebuilt (check this by comparing timestamps of `library/libmbedtls.a` with the installed one
 - `make` - when compiling a project:
   - creates dependency graph (which files depend on other files - for example  `foo.o` depends on `foo.c` and `foo.bin` depends on `foo.o` and `bar.o`)
   - `make` decides which parts of the dependency graph need to be rebuilt based on file modification times (if the target is older than the source - rebuild!)
   - as binaries linked against mbedtls will depend on `libmbedtls.a` - and it's timestamp updates with each compilation - they will always be (unnecessarily) re-linked

Reply:
alternative approach would be to not use `install` target at all but just to copy the needed files by hand

Reply:
Thanks for explaining that, firstly I didn't notice that there is `P` not `p`. I hope that `RPp` arguments will be ok then